src/main/java/com/example/carservice/controller/CarController.java

```diff
@@ -13,13 +12,20 @@
 public class CarController {
 
     private final CarService service;
+    private final CarRepository carRepository;
 
-    public CarController(CarService service) {
+    public CarController(CarService service, CarRepository carRepository) {
         this.service = service;
+        this.carRepository = carRepository;
     }
 
     @GetMapping
     public List<Car> getCars() {
         return service.getCars();
     }
+
+    @PutMapping("update")
```

Comment:
In mapping you don't need to use action name. Just write without anything @PutMapping.